src/main/java/com/launchcode/recipeproject/controllers/AuthController.java

```diff
@@ -16,7 +16,6 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.validation.Valid;
 
-@Controller
```

Comment:
Are you sure you should be getting rid of that?

Reply:
I don't know why that was deleted. On now.